device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -282,7 +282,7 @@ public void onMessageReceived(IotHubTransportMessage message, Throwable e)
         }
         else if (message != null)
         {
-            log.info("Message was received from IotHub ({})", message);
+            log.debug("Message was received from IotHub ({})", message);
```

Comment:
This could overwhelm logs while handling with a huge amount of messages, so move logging level from INFO to DEBUG